services/app-api/forms/mcpar.json

```
@@ -4414,7 +4406,7 @@
                 "validation": "number",
                 "props": {
                   "label": "D1.XIII.4 Percentage of standard PA requests that were approved",
-                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were approved. If you choose not to respond prior to June 2026, enter \"NR\" for not reporting.",
+                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were approved. If you choose not to respond prior to June 2026, enter “NR“ for not reporting.",
```

fix double open quote
```suggestion
                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were approved. If you choose not to respond prior to June 2026, enter “NR” for not reporting.",
```